SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -19,12 +19,17 @@ DEFINE_LOG_CATEGORY(LogSpatialWorkerConnection);
 
 namespace
 {
-SpatialGDK::ComponentData ToComponentData(FWorkerComponentData* Data)
+SpatialGDK::ComponentData ToComponentData(const FWorkerComponentData* Data)
 {
 	return SpatialGDK::ComponentData(SpatialGDK::OwningComponentDataPtr(Data->schema_type), Data->component_id);
 }
 
-SpatialGDK::ComponentUpdate ToComponentUpdate(FWorkerComponentUpdate* Update)
+SpatialGDK::ComponentData ToComponentDataRef(const FWorkerComponentData& Data)
```

Comment:
I'm not sure what this means

Reply:
If it's just the same but taking a reference I'd say it's misleading because the FWorkerComponentData is an "owning" pointer. And we should just get rid of FWorkerComponentData from the codebase if we want to clean this up (or augment it to be a ComponentData if it's still useful for some reason)

Reply:
Yeah it's an overload that takes a reference - if it actually used the same name, overload resolution in Algo::Transform can't deduce a type for the overloaded function. 
Will address.